code/game/machinery/constructable_frame.dm

```diff
@@ -846,6 +846,14 @@ to destroy them and players will be able to make replacements.
 							/obj/item/stack/cable_coil = 1,
 							/obj/item/stack/sheet/glass = 4)
 
+/obj/item/circuitboard/cell_charger
+	board_name = "Cell Charger"
+	build_path = /obj/machinery/cell_charger
+	board_type = "machine"
+	origin_tech = "powerstorage=3;materials=2"
+	req_components = list(
+							/obj/item/stock_parts/capacitor = 1)
```

Comment:
```suggestion
	req_components = list(/obj/item/stock_parts/capacitor = 1)
```

Reply:
should maybe refractor the whole file because it's formatted like that, but I'll change it to the way you suggested